src/split/splitBed.cpp

```diff
@@ -148,7 +148,7 @@ std::FILE* BedSplit::saveFileChunk(std::string& filename,size_t file_index)
     
     char tmp[10];
     filename.assign(this->outfileprefix);
-    sprintf(tmp,"%05d",(file_index+1));
+    sprintf(tmp,"%05lu",(file_index+1));
```

Comment:
`%05zu`; but probably it would be best to just avoid the issue by building the filename via an `ostringstream`.